### PR TITLE
release blog post for 6.1

### PR DESCRIPTION
**Pull request by daniellee**

### CODE REVIEW

File: content/blog/2019-04-02-grafana-v6-1-released.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
++++
+
+# v6.1 Stable released!
+
+It has been a few weeks since the excitement of the major Grafana 6.0 release and GrafanaCon which means it is time for a new Grafana release. Grafana 6.1 iterates on the permissions system to allow for teams to be more self-organizing. It also includes a feature for Prometheus that enables a more exploratory workflow for dashboards.
+
+## What's New in Grafana v6.1
+
+- [Ad hoc Filtering for Prometheus]({{< relref "#ad-hoc-filtering-for-prometheus" >}}) - A new query focused workflow for ad-hoc data exploration and troubleshooting.
+- [Permissions: Editors can own dashboards, folders and teams they create]({{< relref "#editors-can-own-dashboards-folders-and-teams-they-create" >}}) - Integration with the new open source log aggregation system from Grafana Labs.
```
Comment (torkelo) [inline content/blog/2019-04-02-grafana-v6-1-released.md]: remove: Integration with the new open source log aggregation system from Grafana Labs.

File: content/blog/2019-04-02-grafana-v6-1-released.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+### Minor Features and Fixes
+
+This release contains a lot of small features and fixes:
+
+- A new keyboard shortcut `d l` toggles all Graph legends in a dashboard.
+- A small bug fix for Elasticsearch - template variables in the alias field now work properly.
+- Some new capabilities have been added for datasource plugins that will be of interest to plugin authors:
+  - a new oauth pass-through option.
+  - it is now possible to add user details to requests sent to the dataproxy.
+- Heatmap and Explore fixes.
```
Comment (davkal) [inline content/blog/2019-04-02-grafana-v6-1-released.md]: I'd add the prom query alignment here:
```
- The Prometheus range query alignment was moved down by one interval. If you have added an `offset` to your queries to compensate for alignment issues, you can now safely remove it.
```

**Comment by tomwilkie** [comment]
Spoke with Daniel, we'll aim to publish this on 3rd (tomorrow)

**Comment by daniellee** [comment]
@tomwilkie @jkld11 it gets published when the release is done but it looks very likely that it will be tomorrow.

**Pull request merged**